src/features/locate_me/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Locate24 } from '@konturio/default-icons';
+import {
+  currentMapPositionAtom,
+  currentNotificationAtom,
+  toolbarControlsAtom,
+} from '~core/shared_state';
+import { controlGroup, controlVisualGroup } from '~core/shared_state/toolbarControls';
+import { i18n } from '~core/localization';
+import { LOCATE_ME_CONTROL_ID, LOCATE_ME_CONTROL_NAME } from './constants';
+
+function successCb(location: GeolocationPosition) {
+  const { coords } = location;
+  const { latitude: lat, longitude: lng } = coords;
+
+  currentMapPositionAtom.setCurrentMapPosition.dispatch({ lat, lng, zoom: 14 });
+  disableControl();
+}
+
+function errorCb(error: GeolocationPositionError) {
+  currentNotificationAtom.showNotification.dispatch(
+    'warning',
+    { title: i18n.t('locate_me.get_location_error' + ': ' + error.message) },
```

Comment:
https://wordpress.org/support/topic/i18n-please-dont-build-sentences-by-concatenating-substrings/

Reply:
also, error.message gets untranslated?

Reply:
it's not clear what's going on here, could you please explain? @IlyaIzr 
btw we shouldn't pass dynamic stuff to i18n.t, as it can't work with future i18n static analyser